app/uk/gov/hmrc/incometaxsubscriptionselfemployedfrontend/views/SelfEmployedCYA.scala.html

```diff
@@ -82,7 +82,7 @@
     simpleSummaryListRow(
         key = messages("self-employed-cya.business-name"),
         value = businessName,
-        changeHref = uk.gov.hmrc.incometaxsubscriptionselfemployedfrontend.controllers.individual.routes.BusinessNameController.show(businessId, isEditMode = true).url,
+        changeHref = uk.gov.hmrc.incometaxsubscriptionselfemployedfrontend.controllers.individual.routes.BusinessNameController.show(businessId, isEditMode = true, isGlobalEdit = true).url,
```

Comment:
```suggestion
        changeHref = uk.gov.hmrc.incometaxsubscriptionselfemployedfrontend.controllers.individual.routes.BusinessNameController.show(businessId, isEditMode = true, isGlobalEdit = isGlobalEdit).url,
```

All of these change links should be using a provided isGlobalEdit flag into the view as you don't know without that flag if the user is actually in a global edit state.